### ISSUE TITLE
build failed

### ISSUE THREAD

**Issue by xumoyan** [Moya/Moya#1196]
```swift
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'MoyaTest' do
  # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
  use_frameworks!

  # Pods for MoyaTest
  pod 'Moya', '~> 8.0.5'
  target 'MoyaTestTests' do
    inherit! :search_paths
    # Pods for testing
  end

  target 'MoyaTestUITests' do
    inherit! :search_paths
    # Pods for testing
  end

end
```
in NetworkLoggerPlugin.swift 19 line error：
Static method 'reversedPrint(_:terminator:items:)' is fileprivate and cannot be referenced from a default argument value
I just finished pod...
I don't know what happened
xcode version 9.0 beat

**Comment by sunshinejr** [comment]
Hey @xumoyan. Thanks for the issue! However, this is a duplicate of #1189. Please see it as there is more info about this and current workaround. Basically you created a Swift 4.0 project, and we currently do not support it. You need to change Swift version of Moya to 3.2, or wait for 9.0 release (after #1193 merges).

Let me know if it resolves your problem :)

**Comment by xumoyan** [comment]
thanks,i resolves the problem.
